Lib/ftplib.py

```diff
@@ -713,28 +713,13 @@ class FTP_TLS(FTP):
         '221 Goodbye.'
         >>>
         '''
-        ssl_version = ssl.PROTOCOL_TLS_CLIENT
 
         def __init__(self, host='', user='', passwd='', acct='',
-                     keyfile=None, certfile=None, context=None,
```

Comment:
When you remove positional parameters, you have to make the following parameters keyword-only.

Reply:
Oh right, done. I didn't document this change. Is it worth to document it?

Reply:
Were similar changes documented in the past?

Please add tests that only such positional arguments are accepted. In most cases `None` is a valid argument both for the first deleted parameter and for the first remaining parameter, so you can just add `None` as an excessive positional argument.

Reply:
> Were similar changes documented in the past?

In commit 9baf242fc733ab8a52a0b6201d95c6fdb8251745, I removed a buffering parameter in bz2.BZ2File. I see that I wrote ".. versionchanged:: 3.9 (...) The compresslevel parameter became keyword-only." in the doc.

Honestly, I don't think that it should be documented. I don't expect people to call such constructor with 6 parameters or more. I'm only aware of people who called CodeType() with tons of arguments: problem solved with CodeType.replace() method addition.